src/pages/EditProfile.js

```diff
@@ -36,6 +15,38 @@ setDescrip = (value) => {
     isInEditMode:false
   }
 
+addInfo = (name,info,description)=>{
+  firebase.database().ref('users/').push({
+    name:name,
+    info:info,
+    description:description
+  }).then(()=> Actions.profile());
+  
+};
+
+handleChangeName = e => {
+  this.setState({
+    name: e.nativeEvent.text,
+  });
+};
+
+handleChangeInfo = e => {
+  this.setState({
+    info:e.nativeEvent.text,  
+  });
+};
+
+handleChangeDes = e => {
+  this.setState({
+    description:e.nativeEvent.text
+  });
+};
+
+handleSubmit = () => {
+  this.addInfo(this.state.name,this.state.info,this.state.description);
```

Comment:
use destructuration to have a clearer view of our code

```suggestion
  const { name, info, description } = this.state;
  
  this.addInfo(name, info, description);
```